include/eve/module/real/polynomial/detail/newton_impl.hpp

```diff
@@ -0,0 +1,125 @@
+#pragma once
```

Comment:
This file need a header block and remvoe the comments ...

Reply:
done